src/platform/site-wide/sass/modules/_m-facilities.scss

```diff
@@ -0,0 +1,57 @@
+// Facilities Hub
+.facility-main-cta {
```

Comment:
@acabouet has a PR that also has some facility styles, but the prefixes are different. Can we come up with a consistent pattern for the class names in this file? (cc @ahay-agile6)